widgets/serializers/rss.py

```diff
@@ -56,8 +57,13 @@ class RssFeedWidgetSerializer(WidgetInstanceSerializer):
 
     def get_json(self, instance):
         """Obtains RSS feed data which will then be provided to the React component"""
-        rss = _fetch_rss(instance.configuration["url"])
-        entries = getattr(rss, "entries", [])
+        try:
+            rss = _fetch_rss(instance.configuration["url"])
+            entries = getattr(rss, "entries", [])
+        except:  # pylint: disable=bare-except
+            # log an exception and coerce this to an empty list so the feed and UI don't crash
+            log.exception("Error trying to refresh cached RSS feed")
```

Comment:
would it be good to put the objects ID in the log message here, so we can track it down easily if we see it in the logs?

Reply:
Done, also updated the log message when we save to be more informative too